packages/formula-editor/src/suggestion-menu.d.ts

```
@@ -0,0 +1,8 @@
+import { LitElement } from "lit";
+export declare class SuggestionMenu extends LitElement {
+    recommendations: string[];
+    onClickRecommendation: Function;
```

Biome (1.9.4)

[error] Don't use 'Function' as a type.

Prefer explicitly define the function shape. This type accepts any function-like value, which can be a common source of bugs.

(lint/complexity/noBannedTypes)